rewrite-maven/src/test/java/org/openrewrite/maven/AddOrUpdateDependencyManagementTest.java

```diff
@@ -0,0 +1,310 @@
+package org.openrewrite.maven;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.openrewrite.Change;
+import org.openrewrite.Refactor;
+import org.openrewrite.maven.AddOrUpdateDependencyManagement;
+import org.openrewrite.maven.MavenParser;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class AddOrUpdateDependencyManagementTest {
+    @Test
```

Comment:
Our typical format for tests is to use Kotlin (basically just for the multi-line string support) with some convenience/conventions for testing refactoring visitors. For example, [AddDependencyTest.kt](https://github.com/openrewrite/rewrite/blob/master/rewrite-maven/src/test/kotlin/org/openrewrite/maven/AddDependencyTest.kt). 

Would you consider rewriting these tests to be consistent with this format?

Reply:
Added the test using Kotlin, removed the Java test. Preferred not to write temp files. Hope that's fine.

Reply:
Agreed with not writing to temp files 👍 . I believe the tests that still do that reflect an earlier version of the API before we made sure everything could be done in memory.